submissions/onyemeMichael/to-do-app/src/components/TodoList.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect} from 'react';
+import TodoForm from './TodoForm';
+import Todo from './Todo';
+
+function TodoList() {
+  const [todosLeft,setTodosRemaining] = useState(0);
+  const [todos, setTodos] = useState([]);
+  useEffect(() => {
```

Comment:
Format this block of code

Reply:
Kindly explain further.

Reply:
There is an empty line on 10 and the content of line 12 should be just after the comma on line 12 @MYKEL7